website/blog/why-was-my-bazel-build-so-slow.md

```diff
@@ -0,0 +1,208 @@
+---
+slug: debugging-slow-bazel-builds
+title: Why is my Bazel build so slow?
+author: Maggie Lou
+author_title: "Software Engineer @ BuildBuddy"
+date: 2024-3-14 12:00:00
+author_url: https://www.linkedin.com/in/maggie-lou-4a3bb110a/
+author_image_url: https://avatars.githubusercontent.com/u/13951661
+image: /img/slow.png
+tags: [debugging, product, engineering]
+---
+
+The promise of Bazel is the promise of fast builds. So what do you do if your build was slow? Curse out the developer that convinced your company to migrate to Bazel? Of course not! You’d never do that, right? 🙂
+
+BuildBuddy is here to help. Bazel provides a lot of helpful information to debug slow builds, but it can be overwhelming to know where to look.
+
+Maybe you ran two builds and you’re surprised the second build ran so slowly, when you’d expected it to be cached. Or maybe you ran one build, and you just want to understand where all the time was spent. Here are a couple good places to start, with some real-world examples of why our own builds were slow.
+
```

Comment:
Mind throwing a `<!-- truncate -->` tag in here somewhere so only a snippet shows up in the blog homepage?